data-processing/models/models.py

```diff
@@ -175,5 +206,19 @@ def create_tables() -> None:
             EntityBoundingBox,
             Annotation,
         ],
+        # Don't create the tables if they're already created.
         safe=True,
     )
+
+    output_database.execute_sql(
```

Comment:
@codeviking do you have any pointers on best practices for programmatically granting privileges on automatically-created tables? Will this do or are there gotchas to it?

I also tried commands like 

```sql
ALTER DEFAULT PRIVILEGES
    GRANT SELECT ON TABLES 
TO api;
```

So that the commands would only have to be run once, in a management console. However, that doesn't seem to have worked (maybe I just need to dig deeper into the causes of failure).

Reply:
Nope, the explicit route I think is the best one.

The gotcha is the user executing these will have to have permission to do so.  Provided they created the schema, this should work (I think).

In this case given the data isn't terribly sensitive, you might be able to just do:

```
GRANT ALL ON SCHEMA %s TO %s
```

It's certainly not following the principle of least privilege, but it's ok given the low data sensitivity.